src/main/scala/mesosphere/marathon/core/launcher/InstanceOp.scala

```diff
@@ -36,6 +36,23 @@ object InstanceOp {
     }
   }
 
+  case class LaunchTaskGroup(
+      executorInfo: MesosProtos.ExecutorInfo,
+      groupInfo: MesosProtos.TaskGroupInfo,
+      stateOp: InstanceStateOp,
+      oldInstance: Option[Instance] = None,
+      offerOperations: Iterable[MesosProtos.Offer.Operation]) extends InstanceOp {
+
+    override def applyToOffer(offer: MesosProtos.Offer): MesosProtos.Offer = {
+      import scala.collection.immutable
+      import scala.collection.JavaConverters._
+      val taskResources: immutable.Seq[MesosProtos.Resource] =
```

Comment:
why not use JavaConversions and make it all "nice and simple"? Also, prefer top level imports - and import scala.collection.immutable.Seq explicitly.